cypress/fixtures/projects.json

```diff
@@ -3,7 +3,7 @@
     "id": "6d5fff43-d691-445d-a41a-7d0c639080e6",
     "numIssues": 73,
     "numEvents24h": 7,
-    "status": "error",
+    "status": "critical",
```

Comment:
The changes in this file are likely the problem. This represents the API data. And neither `critical` nor `stable` exist in the API response. So if you revert these changes I guess the Cypress tests will work as well